package.json

```diff
@@ -69,6 +69,7 @@
     "react-emoji-render": "1.2.4",
     "react-i18next": "10.11.4",
     "react-linkify": "1.0.0-alpha",
+    "react-markdown": "^5.0.3",
```

Comment:
Please pin the dependency to the specific version, no caret.

Reply:
Also make sure you check in the changes to package-lock

Reply:
Yes I removed the caret and added the changes in the package-lock.json.

Just out of curiosity, what is the reason that we keep the package-lock.json on the repository rather than running `npm install`?